go/stream/message.go

```diff
@@ -56,6 +56,11 @@ type MessageProperties struct {
 	EncryptionKeyID string `json:"encryptionKeyID,omitempty"` // optional
 }
 
+func (m MessageProperties) String() string {
+	return fmt.Sprintf("RequestType: %s, RoutingKey: %s, WorkspaceID: %s, SourceID: %s, ReceivedAt: %s, RequestIP: %s, DestinationID: %s, UserID: %s, SourceJobRunID: %s, SourceTaskRunID: %s, TraceID: %s, SourceType: %s, WebhookFailureReason: %s, Stage: %s, Compression: %s, Encryption: %s, EncryptionKeyID: %s",
```

Comment:
`fmt.Sprintf` uses reflection of course so we're just moving it from one place to another :sweat_smile:

Reply:
Why not have a method that returns a `[]logger.Field`?

```go
func (m MessageProperties) LoggerFields() []logger.Field {
    n := 0 // use exact number here
    fields := make([]logger.Field, 0, n)
    fields = append(fields, logger.NewStringField(m.RequestType))
    // ...
    return fields
}
```

Reply:
yeah, this seems better. I was also feeling `String()` fn was too generic.